xml/System.Web.Services.Protocols/HtmlFormParameterWriter.xml

```diff
@@ -122,7 +122,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Web.Services.Protocols.HtmlFormParameterWriter.WriteRequest%2A> method calls the <xref:System.Web.Services.Protocols.UrlEncodedParameterWriter.Encode%2A> writer,object[] values)?qualifyHint=True&autoUpgrade=True method.  
+ The <xref:System.Web.Services.Protocols.HtmlFormParameterWriter.WriteRequest%2A> method calls the <xref:System.Web.Services.Protocols.UrlEncodedParameterWriter.Encode(System.IO.TextWriter,System.Object[])?displayProperty=nameWithType> method.  
```

Comment:
given that the original was an autoUpgrade=true (which in the old days meant you wanted to link to the overload page),  you want to link to the * version

so change it to System.Web.Services.Protocols.UrlEncodedParameterWriter.Encode%2A?displayProperty=nameWithType